os.kieserver.webapp/configure.sh

```diff
@@ -61,6 +61,19 @@ chmod 664 "${KIE_SERVER_WEBINF_DIR}/web.xml"
 if [[ "$JBOSS_EAP_VERSION" == "6."* ]]; then
     cp -f -p ${ADDED_METAINF_DIR}/kie-server-jms-eap6x.xml ${KIE_SERVER_METAINF_DIR}/kie-server-jms.xml
     cp -f -p ${ADDED_WEBINF_DIR}/jboss-deployment-structure-eap6x.xml ${KIE_SERVER_WEBINF_DIR}/jboss-deployment-structure.xml
+
+    # https://access.redhat.com/solutions/3637671
+    compiler_patch_name="jboss-bxms-6.4.11-RHBPMS-5222"
```

Comment:
it is expected to have a specific jar version hardcoded?

Reply:
@spolti Yes. This was a headache of a problem that previous to this refactoring, Prod had to spin completely new EAP 6.4 images _just for us_ to include this patch, since we used to build on top of EAP openshift image. Now with this refactoring, we can just apply the patch ourselves. Without this specific patch, kieserver startup blows up. We shouldn't have any more of these, however, since 6.4 is very much in maintenance mode.